geomesa-kafka/geomesa-kafka-utils/src/main/scala/org/locationtech/geomesa/kafka/consumer/ThreadedConsumer.scala

```diff
@@ -53,6 +53,7 @@ abstract class ThreadedConsumer(
     val format = if (consumers.lengthCompare(10) > 0) { "%02d" } else { "%d" }
     var i = 0
     consumers.foreach { c =>
+      c.subscription().asScala.foreach(topics.add)
```

Comment:
What was happening here before?  I don't have the context to know what's up here:).

Reply:
the topics was a lazy val that was evaluated here, but since consumers are aggressively single threaded, it was throwing ConcurrentAccess exceptions once the consumers were started up in separate threads.

Reply:
so we add the topics before starting the separate threads to ensure that only a single thread accesses the consumer at once